runelite-client/src/main/java/net/runelite/client/plugins/timetracking/TimeTrackingPlugin.java

```diff
@@ -172,6 +169,13 @@ else if (clockManager.getStopwatches().isEmpty() && e.getKey().equals(STOPWATCHE
 		{
 			clockManager.loadStopwatches();
 		}
+
+		if (e.getKey().equals(FARM_TICK_OFFSET_RESET) && e.getNewValue().equals("true"))
```

Comment:
`newValue` can be null. In general prefer `"value".equals(thing)` to `thing.equals(value)`